public/pages/mp-page.html

```diff
@@ -1,25 +1,45 @@
 <!DOCTYPE html>
 <script src="https://unpkg.com/react@18/umd/react.development.js"></script>
 <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js"></script>
+<script src="https://unpkg.com/i18next@latest/dist/umd/i18next.js"></script>
+<script src="https://unpkg.com/react-i18next@latest/dist/umd/react-i18next.js"></script>
+<script src="https://unpkg.com/i18next-browser-languagedetector@latest/dist/umd/i18nextBrowserLanguageDetector.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/i18next-http-backend@1.3.1/i18nextHttpBackend.min.js"></script>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>MP Details</title>
+    <title>Is My MP a Landlord? | MP Detail</title>
     <link rel="stylesheet" href="../index.css">
 </head>
 <body>
-    <head><title>Is My MP a Landlord?</title></head>
-    <header class="title pad-bot">
-        <h1><a href="/" class="invis">Is My MP a Landlord?</a></h1>
-        <span>All data sourced from the </span>
-        <a href="https://prciec-rpccie.parl.gc.ca/EN/PublicRegistries/Pages/PublicRegistry.aspx">Office of Conflict of Interest and Ethics Commissioner</a>
-    </header>
     <div>
         <div id="root"></div>
     </div>
-
-    <script src="../js/mp-page.js">
-    </script>
 </body>
-</html>
\ No newline at end of file
+</html>
+<script>
```

Comment:
All elements, including `<script>` elements, should either be in the `<head>` or `<body>` elements which themselves should be within the `<html>` element. HTML is forgiving enough that this works, but this document won’t pass validation for this reason (and numerous others).